src/pages/docs/lexicon/user_card.md

```diff
@@ -0,0 +1,148 @@
+
+_ | Action | Mandatory
+--- | --- | ---
+[1](#step-1) | Locate the code in your application | ✔
+[2](#step-2) | Use `clay:user-card` instead of the old `liferay-frontend:user-vertical-card` tag | ✔
+[3](#step-3) | Move the action items definition into a Display Context | ✘
+
+### [1] Locate the code in your application <a id="step-1"></a>
+
+Locate the code responsible for rendering the **image card** in your application. It should resemble something like the following snippet found in [users-admin/users-admin-web/user/search_columns.jspf](https://github.com/liferay/liferay-portal/blob/master/modules/apps/foundation/users-admin/users-admin-web/src/main/resources/META-INF/resources/user/search_columns.jspf#L49-L59):
+
+```text/html
+<liferay-frontend:user-vertical-card
+	actionJsp="/user_action.jsp"
+	actionJspServletContext="<%= application %>"
+	cssClass="entry-display-style"
+	resultRow="<%= row %>"
+	rowChecker="<%= rowChecker %>"
+	subtitle="<%= user2.getScreenName() %>"
+	title="<%= user2.getFullName() %>"
+	url="<%= String.valueOf(rowURL) %>"
+	userId="<%= user2.getUserId() %>"
+/>
+```
+
+### [2] Use `clay:user-card` instead of the old `liferay-frontend:user-vertical-card` tag <a id="step-2"></a>
+
+Start by adding the following imports to the imports section of your `init.jsp` file:
+
+```text/html
+// Import the clay tld file to be able to use the new tag
+<%@ taglib uri="http://liferay.com/tld/clay" prefix="clay" %>
+
+// Import the DropdownItem utility class to create the items model
+<%@ page import="com.liferay.frontend.taglib.clay.servlet.taglib.util.JSPDropdownItemList" %>
+```
+
+Don't forget to add the dependencies with to the `frontend-taglib-clay` and `fronteng.taglib.soy` module in your `build.gradle` file:
+
+```text/html
+provided group: "com.liferay", name: "com.liferay.frontend.taglib.soy", version: "1.0.0"
+
+provided project(":apps:foundation:frontend-taglib:frontend-taglib-clay")
+```
+
+- Url param is now href.
+- Text param is now title.
+- Icon is passed now trough icon param instead as a content with `horizontal-card-col` and `horizontal-card-col-icon`
+- For the actions you need to model them using the `JSPDropdownItemList` class.
+
+```text/html
+<clay:user-card
+	actionItems="<%=
+		new JSPDropdownItemList(pageContext) {
+				{
+					if (hasUpdatePermission) {
+						add(
+								dropdownItem -> {
+										PortletURL editUserURL = renderResponse.createRenderURL();
+										editUserURL.setParameter("mvcRenderCommandName", "/users_admin/edit_user");
+										editUserURL.setParameter("redirect", redirect);
+										editUserURL.setParameter("p_u_i_d", String.valueOf(userId));
+
+										dropdownItem.setHref(editUserURL);
+										dropdownItem.setLabel(LanguageUtil.get(request, "edit"));
+								}
+						);
+					}
+				}
+		}
+	%>"
+	elementClasses="entry-display-style"
+	href="<%= String.valueOf(rowURL) %>"
+	initials="<%= user2.getInitials() %>"
+	inputValue="<%= user2.getFullName() %>"
```

Comment:
We get inputValue from `resultRow` in the tag, there's no need to pass it when you pass  `resultRow`